pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -926,7 +926,9 @@ def test_setitem_duplicate_columns_not_inplace(self):
         tm.assert_frame_equal(df_view, df_copy)
         tm.assert_frame_equal(df, expected)
 
-    @pytest.mark.parametrize("value", [1, np.array([[1], [1]]), [[1], [1]]])
+    @pytest.mark.parametrize(
+        "value", [1, np.array([[1], [1]], dtype="int64"), [[1], [1]]]
```

Comment:
why?

this is causing me problems in #35417 and i thinking making this change would fix that, but would feel like cheating

Reply:
This is just adding a fixed dtype, doesn't change anything else. The default dtype of numpy is platform dependent, while in pandas it's always int64 (as you know ;))